controllers/reconcilers/kserve_istio_servicemonitor_reconciler.go

```diff
@@ -33,23 +32,24 @@ const (
 	istioServiceMonitorName = "istiod-monitor"
 )
 
+var _ SubResourceReconciler = (*KserveIstioServiceMonitorReconciler)(nil)
+
 type KserveIstioServiceMonitorReconciler struct {
```

Comment:
This class should implement `SingleResourcePerNamespace`

Reply:
Done 28944bc